go.mod

```diff
@@ -4,15 +4,12 @@ go 1.12
 
 require (
 	github.com/ethereum/go-ethereum v1.9.9
-	github.com/go-stack/stack v1.8.0 // indirect
 	github.com/gogo/protobuf v1.1.1
 	github.com/howeyc/fsnotify v0.9.0 // indirect
 	github.com/jmoiron/sqlx v0.0.0-20181024163419-82935fac6c1a
 	github.com/magiconair/properties v1.8.0
-	github.com/makerdao/vulcanizedb v0.0.13-rc.2
+	github.com/makerdao/vulcanizedb v0.0.13-rc.2.0.20200309155515-0dcc44e856be
```

Comment:
will switch back to staging and/or a release if we get it merged on the vdb side